requirements.in

```diff
@@ -0,0 +1,27 @@
+astunparse
```

Comment:
Can you please explain is this requirement file is for bazel build or runtime?

Reply:
yes, it defines hermetic python environment within bazel

Reply:
Thanks for the explanation, I think PyTorch/XLA only use bazel to build a shared library [here](https://github.com/pytorch/xla/blob/master/BUILD#L7), bazel test targets also don't have any python dependencies.(Bazel test is only used for cpp tests) Therefore, maybe we can remove all of these? (Or at least something like `google-api-python-client`, `google-cloud-storage`, I guess we definitely don't need those in the bazel build)